cmd/object-handlers.go

```diff
@@ -319,7 +334,6 @@ func (api objectAPIHandlers) CopyObjectHandler(w http.ResponseWriter, r *http.Re
 	vars := mux.Vars(r)
 	dstBucket := vars["bucket"]
```

Comment:
CopyObject should also disable caching when hasSSECustomerHeader is set. same as CopyObjectPart()